### DISCUSSION TITLE
Ane for Google Play Billing Library Version 3

### DISCUSSION THREAD

**Discussion by hotdes** [airsdk/Adobe-Runtime-Support#1552] [Q&A]
Google Play currently requires Google Play Billing Library version 3.
ANE I bought 5 years ago  uses Google Play Billing Library version 2.
What to do?
Are there such ANE for  Google Play Billing Library version 3?

**Comment by apofis1969** [comment]
Hi,
have a look at https://airnativeextensions.com/extensions
There you find what you need.
